package.json

```diff
@@ -38,18 +38,20 @@
     "clean:images": "yarn workspace @itwin/itwinui-css clean:images",
     "createComponent:react": "node scripts/createComponent.js",
     "push-docker:react": "yarn workspace storybook push-docker",
-    "clean": "turbo run clean && rimraf node_modules"
+    "clean": "turbo run clean && rimraf node_modules",
+    "release": "yarn build --filter={./packages/*}... && changeset publish"
   },
   "dependencies": {},
   "devDependencies": {
+    "@changesets/cli": "^2.26.0",
     "chokidar-cli": "^3.0.0",
     "concurrently": "^7.6.0",
+    "cross-env": "^7.0.3",
     "husky": "^8.0.2",
     "lint-staged": "^12.5.0",
     "prettier": "2.8.0",
     "rimraf": "^3.0.2",
-    "turbo": "~1.6.3",
-    "cross-env": "^7.0.3"
+    "turbo": "~1.6.3"
```

Comment:
This change can be removed

Reply:
no need. this was automatically sorted when i ran `yarn add` to add changesets